tests/should-fail/iati-activities/iati-activity/result/document-link/title/narrative/err-not-iati-xml-missing-required-element_no-narrative-element.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity>
+    <iati-identifier></iati-identifier>
+    <reporting-org type="xx" ref="xx">
+      <narrative>Organisation name</narrative>
+    </reporting-org>
+    <title>
+      <narrative>Xxxxxxx</narrative>
+    </title>
+    <description>
+      <narrative>Xxxxxxx</narrative>
+    </description>
+    <participating-org role="xx"></participating-org>
+    <activity-status code="xx"/>
+    <activity-date type="xx" iso-date="2013-11-27"/>
+    <activity-date type="xx" iso-date="2013-11-27">
+      <narrative>Some stuff ere</narrative>
+    </activity-date>
+    <result type="xx">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <document-link url="xx" format="xx">
+        <title>
+          <!-- FAIL: Required `title` element added, but without the required 'narrative' child element. -->
```

Comment:
Remove whitespace

Reply:
Is the suggestion that this should be:

```
<title/><!-- FAIL: Required `title` element added, but without the required 'narrative' child element. -->
```

??